src/components/Catalog/FilterSidebar/FilterSidebar.tsx

```diff
@@ -17,12 +17,19 @@ import {
   GameLanguage,
 } from "../../../types/Game";
 import { TranslationLabelType } from "../../../utils/translations.ts";
-import { TLabel } from "../../ui/TranslationLabel/TLabel";
+import {
+  getTranslatedText,
+  TLabel,
+  TText,
+} from "../../ui/TranslationLabel/TLabel";
 
-import Dropdown, { DropdownOption } from "./Dropdown/Dropdown";
+import Dropdown from "./Dropdown/Dropdown";
 
 import "./FilterSidebar.css";
 
+import { TranslationContext } from "@/context/TranslationProvider.tsx";
+import { capitalizeFirstLetter } from "@/utils/capitalizeFirstLetter.ts";
```

Comment:
```suggestion
import { TranslationContext } from "@context/TranslationProvider.tsx";
import { capitalizeFirstLetter } from "@utils/capitalizeFirstLetter.ts";
```
**Suggestion** : Remet dans l'ordre les imports